### ISSUE TITLE
Compile for `armv6-linux-guneabihf`

### ISSUE THREAD

**Issue by cocoa-xu** [cocoa-xu/libedgetpu#1]

**Comment by cocoa-xu** [comment]
An experimental `armv6-linux-guneabihf` version is available in [v0.1.3](https://github.com/cocoa-xu/libedgetpu/releases/tag/v1.0.3), following the steps [here](https://github.com/google-coral/edgetpu/issues/229#issuecomment-751504398) plus some other hacks.

However, it's based on the original FrogFish release, 14eee1a076aa1af7ec1ae3c752be79ae2604a708, which was released in Nov 5, 2020. So it's highly likely that there will be some issues using it with latest tensorflow lite version (v2.11.0, as of the time of writing this).

**Comment by cocoa-xu** [comment]
No luck getting it compiled for `armv6-linux-gnueabihf`, see workflow, https://github.com/cocoa-xu/libedgetpu/actions/runs/4397878453/jobs/7701321126.

```
ERROR: /home/cocoa/.cache/bazel/_bazel_cocoa/eab0d61a99b6696edb3d2aff87b585e8/external/com_google_absl/absl/base/BUILD.bazel:152:11: Compiling absl/base/internal/low_level_alloc.cc failed: (Exit 1): process-wrapper failed: error executing command 
  (cd /home/cocoa/.cache/bazel/_bazel_cocoa/eab0d61a99b6696edb3d2aff87b585e8/sandbox/processwrapper-sandbox/16/execroot/libedgetpu && \
  exec env - \
    PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
    PWD=/proc/self/cwd \
    TMPDIR=/tmp \
  /home/cocoa/.cache/bazel/_bazel_cocoa/install/ec7bc80570c57e35507fafa6aadd57b1/process-wrapper '--timeout=0' '--kill_delay=15' /usr/bin/arm-linux-gnueabihf-gcc -MD -MF bazel-out/armv6-opt/bin/external/com_google_absl/absl/base/_objs/malloc_internal/low_level_alloc.pic.d '-frandom-seed=bazel-out/armv6-opt/bin/external/com_google_absl/absl/base/_objs/malloc_internal/low_level_alloc.pic.o' -fPIC -iquote external/com_google_absl -iquote bazel-out/armv6-opt/bin/external/com_google_absl '-march=armv6' '-mfpu=vfp' -mthumb -mthumb-interwork '-march=armv6' '-mfpu=vfp' -mthumb -mthumb-interwork -Wall -Wextra -Wcast-qual -Wconversion-null -Wmissing-declarations -Woverlength-strings -Wpointer-arith -Wundef -Wunused-local-typedefs -Wunused-result -Wvarargs -Wvla -Wwrite-strings -Wno-missing-field-initializers -Wno-sign-compare -DNOMINMAX -c external/com_google_absl/absl/base/internal/low_level_alloc.cc -o bazel-out/armv6-opt/bin/external/com_google_absl/absl/base/_objs/malloc_internal/low_level_alloc.pic.o)
In file included from /usr/arm-linux-gnueabihf/include/endian.h:60,
                 from /usr/arm-linux-gnueabihf/include/sys/types.h:193,
                 from external/com_google_absl/absl/base/internal/low_level_alloc.h:27,
                 from external/com_google_absl/absl/base/internal/low_level_alloc.cc:20:
/usr/arm-linux-gnueabihf/include/bits/byteswap.h: In function '__uint16_t __bswap_16(__uint16_t)':
/usr/arm-linux-gnueabihf/include/bits/byteswap.h:34:29: sorry, unimplemented: Thumb-1 hard-float VFP ABI
 __bswap_16 (__uint16_t __bsx)
                             ^
Target //tflite/public:libedgetpu_direct_all.so.stripped failed to build
INFO: Elapsed time: 4.990s, Critical Path: 0.44s
INFO: 135 processes: 133 internal, 2 processwrapper-sandbox.
FAILED: Build did NOT complete successfully
make: *** [Makefile:105: libedgetpu-direct] Error 1
make: Leaving directory '/workspace'
make: *** [Makefile:194: docker-build] Error 1
```

**Comment by cocoa-xu** [comment]
Solved in #2
